src/components/data_search/DatasetSearchTable.js

```diff
@@ -148,7 +148,7 @@ export const DatasetSearchTable = (props) => {
               truncate: true,
             },
             {
-              value: entry[0].study.dataCustodianEmail.join(', '),
+              value: entry[0].study.dataCustodianEmail?.join(', '),
```

Comment:
I think this will still error out. This will pass `undefined` to `join` which will throw. I think we need a conditional here to avoid trying to send anything invalid to `join`

Reply:
See previous comment, according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining , it will short circuit and not run the join if the email is undefined.

Reply:
Minimal example:

```
a = { b: [ 'c', 'd' ], e: undefined };
a.b.join(', ') // "c, d"
a.e.join(', ') // undefined
```

Reply:
Minimal example:

```
a = { b: [ 'c', 'd' ], e: undefined };
a.b.join(',')  // "c,d"
a.e?.join(',') // undefined
```